config/targets/MISSION/target.txt

```diff
@@ -0,0 +1,4 @@
+#
+# Required Libraries
+#
+#REQUIRE 'mission_lib.rb'
```

Comment:
This is commented out intentionally as COSMOS ended up with a require lockout otherwise. Left for now to investigate later.